### PR TITLE
refactor: remove redundent code

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

During the refactor I didn't notice that this was redundent and doesn't need to be async anymore.

### Type of change

(Remove other not matching type)

- refactor

### How has this been tested?

Manually

### CODE REVIEW

**Review by reneaaron — APPROVED** [review]
utACK

File: src/common/utils/currencyConvert.ts [hunk 1]
```diff
@@ -27,7 +23,7 @@ export const getFiatValue = async ({
   rate: number;
   currency: CURRENCIES;
 }) => {
-  const fiatValue = await satoshisToFiat({
+  const fiatValue = satoshisToFiat({
```
Comment (lisabaut) [inline src/common/utils/currencyConvert.ts:26]: Oh nice.

This allows an even more simplification:

```TS

export const getFiatValue = ({  // remove async
  amount,
  rate,
  currency,
}: {
  amount: number | string;
  rate: number;
  currency: CURRENCIES;
}) => {
  const fiatValue = Number(amount) * rate); // delete func satoshisToFiat above

  return fiatValue.toLocaleString("en", {
    style: "currency",
    currency,
  });
};
```


And remove the `await` from all places where `getFiatValue`  is used

**Review by lisabaut — COMMENTED** [review]
See my comment below.

This change can be improved IMO

File: src/app/components/AllowanceMenu/index.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
   useEffect(() => {
     if (budget !== "" && showFiat) {
       const getFiat = async () => {
```
Comment (lisabaut) [inline src/app/components/AllowanceMenu/index.tsx:41]: remove async

File: src/app/context/AccountContext.tsx [hunk 3]
```diff
@@ -75,7 +75,7 @@ export function AccountProvider({ children }: { children: React.ReactNode }) {
 
   const updateFiatValue = useCallback(
     async (balance: string | number) => {
```
Comment (lisabaut) [inline src/app/context/AccountContext.tsx:77]: remove async

File: src/app/screens/ConfirmKeysend/index.tsx [hunk 4]
```diff
@@ -49,7 +49,7 @@ function ConfirmKeysend() {
   useEffect(() => {
     (async () => {
```
Comment (lisabaut) [inline src/app/screens/ConfirmKeysend/index.tsx:50]: remove async

File: src/app/screens/ConfirmKeysend/index.tsx [hunk 5]
```diff
@@ -58,7 +58,7 @@ function ConfirmKeysend() {
   useEffect(() => {
     if (showFiat) {
       (async () => {
```
Comment (lisabaut) [inline src/app/screens/ConfirmKeysend/index.tsx:60]: remove async

File: src/app/screens/ConfirmPayment/index.tsx [hunk 6]
```diff
@@ -49,7 +49,7 @@ function ConfirmPayment() {
   useEffect(() => {
     (async () => {
```
Comment (lisabaut) [inline src/app/screens/ConfirmPayment/index.tsx:50]: remove async

File: src/app/screens/ConfirmPayment/index.tsx [hunk 7]
```diff
@@ -58,7 +58,7 @@ function ConfirmPayment() {
...
```
Comment (lisabaut) [inline src/app/screens/ConfirmPayment/index.tsx:59]: remove async

File: src/app/screens/LNURLPay/index.tsx [hunk 8]
```diff
@@ -66,7 +66,7 @@ function LNURLPay() {
   useEffect(() => {
     if (valueSat !== "" && showFiat) {
       const getFiat = async () => {
```
Comment (lisabaut) [inline src/app/screens/LNURLPay/index.tsx:68]: remove async

File: src/app/screens/LNURLWithdraw/index.tsx [hunk 9]
```diff
@@ -45,7 +45,7 @@ function LNURLWithdraw() {
   useEffect(() => {
     if (valueSat !== "" && showFiat) {
       (async () => {
```
Comment (lisabaut) [inline src/app/screens/LNURLWithdraw/index.tsx:47]: remove async

File: src/app/screens/Receive/index.tsx [hunk 10]
```diff
@@ -64,7 +64,7 @@ function Receive() {
   useEffect(() => {
     if (formData.amount !== "" && showFiat) {
       (async () => {
```
Comment (lisabaut) [inline src/app/screens/Receive/index.tsx:66]: remove async

File: src/common/utils/__tests__/currencyConvert.test.ts [hunk 11]
```diff
@@ -4,7 +4,7 @@ import { getFiatValue, getSatValue } from "../currencyConvert";
 
 describe("Currency coversion utils", () => {
   test("getFiatValue", async () => {
```
Comment (lisabaut) [inline src/common/utils/__tests__/currencyConvert.test.ts:6]: remove async

File: src/common/utils/currencyConvert.ts [hunk 12]
```diff
@@ -3,22 +3,18 @@
  */
 import type { CURRENCIES } from "../constants";
 
-const satsToFiat = async (amount: number | string, rate: number) => {
-  return Number(amount) * rate;
-};
-
-const satoshisToFiat = async ({
+const satoshisToFiat = ({
```
Comment (lisabaut) [inline src/common/utils/currencyConvert.ts:6]: As said below, this function is not used elsewhere
you may delete it as it does only `Number(amountInSats) * rate` 
see suggestion code [here](https://github.com/getAlby/lightning-browser-extension/pull/1643/files#r1000299863)

**Review by lisabaut — COMMENTED** [review]
Found some more asyncs 😃

File: src/app/context/AccountContext.tsx (hunk 3 above)
Comment (lisabaut) [inline src/app/context/AccountContext.tsx:77]: async

File: src/app/screens/Home/index.tsx [hunk 13]
```diff
@@ -129,7 +129,7 @@ function Home() {
 
       for await (const payment of payments) {
```
Comment (lisabaut) [inline src/app/screens/Home/index.tsx:130]: Await

File: src/app/screens/Publisher.tsx [hunk 14]
```diff
@@ -85,7 +85,7 @@ function Publisher() {
         });
         for await (const payment of payments) {
```
Comment (lisabaut) [inline src/app/screens/Publisher.tsx:86]: await

**Review by lisabaut — APPROVED** [review]
Thanks, found a few more. See below.

**Pull request merged**